### PR TITLE
BIP-126: Best Practices for Heterogeneous Input Script Transactions

### PR DESCRIPTION
**Pull request by justusranvier**
When a Bitcoin transaction contains inputs that reference previous transaction outputs sent to different Bitcoin addresses, personally identifiable information of the user will leak into the blockchain in an uncontrolled manner. While undesirable, these transactions are frequently unavoidable due the natural
fragmentation of wallet balances over time.

This standard proposes a set of best practice guidelines which minimize the uncontrolled disclosure of personally identifiable information by defining standard forms for transactions containing heterogenous input scripts.

### CODE REVIEW

File: bip-0126.mediawiki [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<pre>
+  BIP: 126
+  Title: Best Practices for Heterogeneous Input Script Transactions
+  Author: Kristov Atlas <kristov@openbitcoinprivacyproject.org>
```
Comment (dcousens) [inline bip-0126.mediawiki:4]: Was this by you or @kristovatlas ?

Comment (justusranvier) [inline bip-0126.mediawiki:4]: I transcribed the results of the discussions that occurred during the OBPP weekly meetings.

Comment (dcousens) [inline bip-0126.mediawiki:4]: Out of interest, are the OBPP weekly meetings open to attend?

Comment (kristovatlas) [inline bip-0126.mediawiki:4]: Yes, I'll follow up with you privately. :-)

**Pull request merged**